### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by heymartinadams**
Duplicate entry of `--`, or missing additional parameter, as the case may be.

### Platforms affected

Docs

### What does this PR do?

Correction

### What testing has been done on this change?

Terminal on mac.

### Checklist
**n/a**
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

**Comment by janpio** [comment]
See the line above your edit:

> You should use double `--` to indicate that these are platform-specific arguments, for example:

**Comment by heymartinadams** [comment]
It wasn’t clear to me from the docs that the empty `--` also had a purpose. Terminal earlier threw an error with it, so I thought that was the cause, but somehow now it doesn’t.

File: www/docs/en/6.x/guide/platforms/android/index.md [hunk 1]
```diff
@@ -310,7 +310,7 @@ To sign an app, you need the following parameters:
 These parameters can be specified using the command line arguments above to
 the [Cordova CLI][cli_reference] `build` or `run` commands.
 
-__Note__: You should use double `--` to indicate that these are platform-specific arguments, for example:
+__Note__: You should use double `--` (followed by a space) to indicate that these are platform-specific arguments, for example:
```
Comment (janpio) [inline www/docs/en/6.x/guide/platforms/android/index.md]: maybe add an "i.e. `-- --`" in the brackets as well.

Comment (janpio) [reply www/docs/en/6.x/guide/platforms/android/index.md]: @heymartinadams Seen my comment?

Comment (heymartinadams) [reply www/docs/en/6.x/guide/platforms/android/index.md]: Oh gosh, I’m such a novice when it comes to GitHub pull requests. How do I do these changes now, @janpio?

Comment (heymartinadams) [reply www/docs/en/6.x/guide/platforms/android/index.md]: I may have deleted my fork?

Comment (heymartinadams) [reply www/docs/en/6.x/guide/platforms/android/index.md]: @janpio See if the other two PRs #763 and #764 do the trick.

Comment (janpio) [reply www/docs/en/6.x/guide/platforms/android/index.md]: They do, but content is strange now.

**Comment by janpio** [comment]
@heymartinadams Are you interested in getting this fixed up and merged?

**Pull request closed without merging**